packages/common/src/directives/ng_optimized_image/ng_optimized_image.ts

```diff
@@ -445,6 +432,17 @@ export class NgOptimizedImage implements OnInit, OnChanges, OnDestroy {
         'disableOptimizedSrcset',
       ]);
     }
+    if (changes['ngSrc'] && !changes['ngSrc'].isFirstChange()) {
+      const oldSrc = this._renderedSrc;
+      this.updateSrcAndSrcset(true);
+      const newSrc = this._renderedSrc;
+      if (this.lcpObserver !== null && oldSrc && newSrc && oldSrc !== newSrc) {
+        const ngZone = this.injector.get(NgZone);
+        ngZone.runOutsideAngular(() => {
+          this.lcpObserver?.recordNgSrcModified(oldSrc, newSrc);
```

Comment:
nit: I'd propose calling this function as `updateImage` for consistency with `registerImage` and `unregisterImage`:
```suggestion
          this.lcpObserver?.updateImage(oldSrc, newSrc);
```

Reply:
Sure, done.